tests/dragonfly/script_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from redis import asyncio as aioredis
+
+lua_script = """
+return "Hello, world!"
+"""
+
+"""
+Test SCRIPT FLUSH command.
+"""
+
+
+@pytest.mark.asyncio
+async def test_script_flush(async_client):
+    sha1 = await async_client.script_load(lua_script)
+    exists = await async_client.script_exists(sha1)
+    assert exists[0] == True
+
+    result = await async_client.script_flush()
+    exists = await async_client.script_exists(sha1)
```

Comment:
In the unit test I would also add a load script after flush and check that it was loaded

Reply:
fix: https://github.com/dragonflydb/dragonfly/pull/2493/commits/af6260015015a3f90745b2d9cc7831af2b2438a8